mcu/apollo4p/am_mcu_apollo.h

```diff
@@ -33,6 +33,9 @@
 // contributors may be used to endorse or promote products derived from this
 // software without specific prior written permission.
 //
+// Third party software included in this distribution is subject to the
```

Comment:
The declaration "
//
// Third party software included in this distribution is subject to the
// additional license terms as defined in the /docs/licenses directory.
"
should be excluded from all files based on the previous marketing alignment.